MaterialDesignThemes.Wpf/HintAssist.cs

```diff
@@ -128,5 +129,31 @@ public static void SetHintOpacity(DependencyObject element, double value)
         }
 
         #endregion
+
+        #region Brushes
+
+        /// <summary>
+        /// The color for highlighted text.
+        /// </summary>
+        public static readonly DependencyProperty HighlightedForegroundProperty = DependencyProperty.RegisterAttached(
+            "HighlightedForeground", typeof(Brush), typeof(HintAssist), new PropertyMetadata(null));
```

Comment:
"Highlighted" is a bit misleading here, since it is already inside of the `HintAssist` class I think it makes more sense to simply name this `Foreground`